src/booleans.js

```diff
@@ -1,62 +1,122 @@
 function negate(a) {
-  // your code here
-};
+  return !a;
+}
 
 function both(a, b) {
-  // your code here
-};
+  //   if (a === true && b === true) {
+  //     return true;
+  //   }
+  //   return false;
+  // }
+  return a === true && b === true;
+}
 
 function either(a, b) {
-  // your code here
-};
+  if (a === true || b === true) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
As above, can be greatly simplified:
```suggestion
  return a || b;
```